boards/DCM/Src/App/App_SendNonPeriodicCanSignals.c

```diff
@@ -0,0 +1,45 @@
+#include "App_SendNonPeriodicCanSignals.h"
+
+#define CLEAR_INV_FAULT_PARAM_ADDRESS (20U)
+#define WRITE_CMD (1U)
+
+void App_SendNonPeriodicCanSignals_ClearInverterFaults(
+    const struct DcmWorld *world)
```

Comment:
think we should just pass can_tx and can_rx into this function, we shouldnt pass in world since we only need can_tx and can_rx

Reply:
done